Frontend/CO.CDP.OrganisationApp/ThirdPartyApiClients/CompaniesHouse/CompaniesHouseApi.cs

```diff
@@ -21,7 +24,7 @@ public class CompaniesHouseApi(IConfiguration configuration, ILogger<CompaniesHo
                 .GetAsync()
```

Comment:
If I search for a valid company number with a trailing whitespace it does not find it. Which is easily done - if you double click on the number on the companies house website in order to select and copy and paste it, it actually comes with a trailing space.

Could we trim the value before searching?